angular/projects/spark-angular/src/lib/components/sprk-tooltip/sprk-tooltip.component.ts

```diff
@@ -63,6 +69,11 @@ export class SprkTooltipComponent implements AfterViewInit, OnChanges {
    */
   @Input()
   triggerIconType = 'question-filled';
+  /**
+   * The vertical alignment of the tooltip.
+   */
+  @Input()
+  verticalAlignment: 'baseline' | 'top' | 'middle' | 'bottom';
```

Comment:
hey @RVMendoza  whats the default setting for the tooltip? As a user I could see it being useful to have that here in the comment. Like something about how this is to override the default setting.

Reply:
I considered giving it a default, but I wanted it to be additive to tooltip. In production it has not vertical alignment, and for many implementations that works.

I want the user to be able to opt-in to vertical alignment only if they need.

Reply:
also added that into the description!